include/boost/thread/v2/thread.hpp

```diff
@@ -23,6 +23,7 @@ namespace boost
     {
 #ifdef BOOST_THREAD_USES_CHRONO
 
+#if !defined BOOST_THREAD_SLEEP_FOR_IS_STEADY
```

Comment:
IIUC, we have now just two variants:
* BOOST_THREAD_SLEEP_FOR_IS_STEADY
* NOT BOOST_THREAD_SLEEP_FOR_IS_STEADY

Could you rework the code to make this more evident.
See other comments how this can be possible

Reply:
Yes, I can definitely do that. I was just trying to minimize code changes for the PR.